packages/terra-collapse/src/Collapse.jsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Arrange from 'terra-arrange';
+import Button from 'terra-button';
+import IconChevronRight from 'terra-icon/lib/icon/IconChevronRight';
+import Toggler from 'terra-toggler';
+import 'terra-base/lib/baseStyles';
+import './Collapse.scss';
+
+const propTypes = {
+  /**
+   * Content in the body of the collapse component that will be expanded or collapsed
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * Sets the text inside of the button when the collapse component is closed
+   */
+  closedButtonText: PropTypes.node.isRequired,
```

Comment:
Updated: https://github.com/cerner/terra-core/pull/475/commits/f4a04402f61a83d24e4f630cd70e8af02911ef2d
If `isIconOnly` prop is set, I'm using the `closedButtonText` as an aria-label on the button. Based on this I updated the `closedButtonText` type from node to string.